bindata/network/ovn-kubernetes/managed/ovnkube-node.yaml

```diff
@@ -40,6 +40,7 @@ spec:
         network.operator.openshift.io/ovnkube-script-lib-hash: "{{.OVNKubeConfigHash}}"
         # prevent blocks when node critical pods get evicted prior to workloads
         cluster-autoscaler.kubernetes.io/enable-ds-eviction: "false"
+        kubectl.kubernetes.io/default-container: ovnkube-controller
```

Comment:
Thanks! Let's do the same for ovnkube-controller also in `bindata/network/ovn-kubernetes/self-hosted/ovnkube-node.yaml` and we're done. Thanks again!